index.bs

```diff
@@ -145,8 +145,8 @@ class AV1CodecConfigurationBox extends FullBox('av1C', version = 0, 0){
 }
 
 aligned (8) class AV1CodecConfigurationRecord {
-  unsigned int (6) reserved = 0;
-  unsigned int (2) OBU_length_mode;
+  unsigned int (7) reserved = 0;
```

Comment:
7 + 1 + 3 = 11?

The 7 should be 4, no? Missed this in the last pass.

Reply:
indeed ...